src/visualization/efmviz/findUsedGenesLevels_efmviz.m

```diff
@@ -0,0 +1,63 @@
+function [gene_id, gene_expr, gene_sig] = findUsedGenesLevels_efmviz(model, exprData, printLevel)
```

Comment:
Same as the other EFMViz specific functions that are derivatives of existing functions.
Please adapt the original files allowing your modifications as optional. Otherwise this makes maintenance difficult.